### DISCUSSION TITLE
Port Xamarin Form to Uno Platform

### DISCUSSION THREAD

**Discussion by ieie1986** [unoplatform/uno#7477] [General]
I have an cross platform application using xamarin forms platform with syncfusion xamarin form component. Now I want to cross compile it to wasm. Is it possible to do that?

**Comment by jeromelaban** [comment]
See the documentation here: https://platform.uno/docs/articles/get-started-dotnet-new.html?q=xamarin%20forms#uno-platform-webassembly-support-for-xamarin-forms
